frontend/packages/shipwright-plugin/console-extensions.json

```diff
@@ -103,82 +103,84 @@
     "properties": {
       "model": {
         "group": "shipwright.io",
-        "version": "v1beta1",
-        "kind": "BuildRun"
+        "version": "v1alpha1",
+        "kind": "BuildStrategy"
       },
-      "flag": "SHIPWRIGHT_BUILDRUN"
+      "flag": "SHIPWRIGHT_BUILDSTRATEGY_V1ALPHA1"
     }
   },
   {
-    "type": "console.navigation/resource-ns",
+    "type": "console.flag/model",
     "properties": {
-      "perspective": "admin",
-      "section": "builds",
-      "id": "shipwright-builds",
-      "name": "%shipwright-plugin~Shipwright Builds%",
       "model": {
         "group": "shipwright.io",
-        "version": "v1alpha1",
-        "kind": "Build"
-      }
-    },
-    "flags": {
-      "required": ["SHIPWRIGHT_BUILD_V1ALPHA1"],
-      "disallowed": ["SHIPWRIGHT_BUILD"]
+        "version": "v1beta1",
+        "kind": "BuildStrategy"
+      },
+      "flag": "SHIPWRIGHT_BUILDSTRATEGY"
     }
   },
   {
-    "type": "console.navigation/resource-ns",
+    "type": "console.flag/model",
     "properties": {
-      "perspective": "admin",
-      "section": "builds",
-      "id": "shipwright-buildruns",
-      "insertAfter": "shipwright-buildruns",
-      "name": "%shipwright-plugin~Shipwright BuildRuns%",
       "model": {
         "group": "shipwright.io",
         "version": "v1alpha1",
-        "kind": "BuildRun"
-      }
-    },
-    "flags": {
-      "required": ["SHIPWRIGHT_BUILDRUN_V1ALPHA1"],
-      "disallowed": ["SHIPWRIGHT_BUILDRUN"]
+        "kind": "ClusterBuildStrategy"
+      },
+      "flag": "SHIPWRIGHT_CLUSTERBUILDSTRATEGY_V1ALPHA1"
     }
   },
   {
-    "type": "console.navigation/resource-ns",
+    "type": "console.flag/model",
     "properties": {
-      "perspective": "admin",
-      "section": "builds",
-      "id": "shipwright-builds",
-      "name": "%shipwright-plugin~Shipwright Builds%",
       "model": {
         "group": "shipwright.io",
         "version": "v1beta1",
-        "kind": "Build"
-      }
+        "kind": "ClusterBuildStrategy"
+      },
+      "flag": "SHIPWRIGHT_CLUSTERBUILDSTRATEGY"
+    }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "shipwright-tablistpage",
+      "section": "builds",
+      "perspective": "admin",
+      "prefixNamespaced": true,
+      "href": "/shipwright.io",
+      "name": "%shipwright-plugin~Shipwright%"
     },
     "flags": {
-      "required": ["SHIPWRIGHT_BUILD"]
+      "required": ["SHIPWRIGHT_BUILD"],
+      "disallowed": ["SHIPWRIGHT_BUILD_V1ALPHA1"]
```

Comment:
I am still not able to see the Shipwright navigation option. It is because of the disallowed flag as both versions are present and enabled on the cluster. IMO for now add an entry for navigation/href and put the required flag as `SHIPWRIGHT_BUILD_V1ALPHA1`. We will update it once v1alpha1 will be removed or disabled.

Reply:
Good catch! Initially thought these flags would be mutually exclusive lol

fixed :)